Organize the presentations page, add content and update links

dorrab opened the pull request:
I have changed to division from recent/previous presentations to case studies/deep dives.
I think this is a better way to present the content.
I have added 2 recents talks by @mstoodle and @Leonardo2718
I have updated the broken links

Commit:
Organize the presentations page, add content and update links
I have changed to division from recent/previous presentations to case studies/deep dives.
I think this is a better way to present the content.
I have added 2 recents talks by @mstoodle and @Leonardo2718
I have updated the broken links

Signed-off-by: Dorra Bouchiha <dorrab@ca.ibm.com>

mstoodle commented on resources/presentations.md:
```diff
@@ -23,22 +23,24 @@ lang: en
-- [*Beyond the Coffee Cup: Leveraging Java Runtime Technologies for the Polyglot.*](http://www.slideshare.net/0xdaryl/javaone-2015-con7547-beyond-the-coffee-cup-leveraging-java-runtime-technologies-for-polyglot?related=1) Daryl Maier, Java One, October 2015.
-- [*A VM is a VM is a VM: The Secret Path to High Performance Multi-Language Runtimes.*](https://www.youtube.com/watch?v=kOnyJurioyw) Mark Stoodley, JVM Languages Summit, August 2015.
+- [*Testing Testarossa - The Eclipse OMR Compiler*](https://youtu.be/D0BCftV6DpE) Leonardo Banderali, August, 2017.
+- [*How does Eclipse OMR compare to LLVM*](http://www.ustream.tv/recorded/105013815) Mark Stoodley, June, 2017.
```
Technically, the title was "Eclipse OMR: How does all this compare to LLVM?"

mstoodle commented on resources/presentations.md:
```diff
@@ -23,22 +23,24 @@ lang: en
-- [*A VM is a VM is a VM: The Secret Path to High Performance Multi-Language Runtimes.*](https://www.youtube.com/watch?v=kOnyJurioyw) Mark Stoodley, JVM Languages Summit, August 2015.
+- [*Testing Testarossa - The Eclipse OMR Compiler*](https://youtu.be/D0BCftV6DpE) Leonardo Banderali, August, 2017.
+- [*How does Eclipse OMR compare to LLVM*](http://www.ustream.tv/recorded/105013815) Mark Stoodley, June, 2017.
+- [*Eclipse OMR Update Tech Talk*](http://www.ustream.tv/recorded/98846650) Mark Stoodley, developerWorks Open, January 2017.
```
I know the talk didn't have a subtitle, but let's put one here to give people an idea what's in the presentation...Something like: "Eclipse OMR Update: compiler component, JitBuilder and projects that use them" ?

mstoodle commented on resources/presentations.md:
```diff
@@ -23,22 +23,24 @@ lang: en
+- [*Testing Testarossa - The Eclipse OMR Compiler*](https://youtu.be/D0BCftV6DpE) Leonardo Banderali, August, 2017.
+- [*How does Eclipse OMR compare to LLVM*](http://www.ustream.tv/recorded/105013815) Mark Stoodley, June, 2017.
+- [*Eclipse OMR Update Tech Talk*](http://www.ustream.tv/recorded/98846650) Mark Stoodley, developerWorks Open, January 2017.
+- [*Eclipse OMR Tech Talk*](https://developer.ibm.com/open/videos/eclipse-omr-tech-talk/) Mark Stoodley, developerWorks Open, July 2016.
```
Do you have a link to the CUSEC presentation that Robert and Kim and John and Shelley went to? I can provide it if not; we should really add it in here too.

mstoodle replied:
Actually, I think it was Andrew <blush>, since Base9 is under his github id...

youngar replied:
Yeah, It was me and Kim who did the presentation

mstoodle reviewed (changes requested):
Thanks for making this update, Dorra! Just a couple small nits to fix.

dorrab commented:
Hello @mstoodle, I can't find the link. If you have it handy, please send it to me. Otherwise I can ask Shelley.

mstoodle commented:
The presentation is here, I believe: https://github.com/youngar/Base9/blob/master/CUSEC%202017%20-%20Buildering%20a%20JIT.pdf

Commit:
Add link to OMR B9
Added the link to OMR B9 slides and updated the titles of the two dW Open talks

dorrab commented:
Hello @mstoodle, addressed your comments above in the second commit. Please review. thank you

mstoodle commented:
Thanks @dorrab!
